tools/pfe-tools/dev-server/demo.css

```
@@ -44,6 +44,57 @@ body.menu-open {
     'menu body';
 }
 
+#main-header {
+  grid-area: head;
+  display: block;
+}
+
+.logo-bar {
+  display: flex;
+  height: var(--pf-demo-header-height, 4.375rem);
+  align-items: center;
+  gap: 1rem;
+  padding-inline-start: var(--pf-c-masthead--PaddingLeft, var(--pf-c-masthead--inset, 24px));
+  padding-inline-end: var(--pf-c-masthead--PaddingRight, var(--pf-c-masthead--inset, 24px));
+  padding-block: 8px;
+  color: white;
+  background: var(--pf-theme--color--accent, #0066cc);
+}
+
+.logo-bar > a {
+  display: flex;
+  align-items: center;
+  color: white;
+  text-decoration: none;
+  font-size: 1.25rem;
+  font-weight: bold;
+  height: 100%;
+}
+
+.logo-bar > a img {
+  height: 100%;
+}
+
+.logo-bar svg {
+  height: 100%;
+}
+
```

Active, focus, and hover state in mobile should be white as well:

```suggestion

.logo-bar #hamburger {
  --pf-c-button--m-plain--Color: #fff;
  --pf-c-button--m-plain--active--Color: #fff;
  --pf-c-button--m-plain--focus--Color: #fff;
  --pf-c-button--m-plain--hover--Color: #fff;
}

```